alpha/alpha.ino

```diff
@@ -28,6 +28,13 @@ void setup() {
   // Start serial command console
   TS_SerialCmd.init();
   TS_SerialCmd.begin();
+
+  TS_RADIO.begin();
+
+  if (!TS_RADIO.wifi_is_connected()) {
+	  TS_RADIO.wifi_connect();
```

Comment:
we don't want to connect on startup right?

Reply:
also checks like `wifi_is_connected` are probably left to be within the wifi_connect function